### PR TITLE
refactor(meta) add meta._SERVER_TOKENS

### PR DESCRIPTION
**Pull request by bungle**
### Summary

Add `kong.meta._SERVER_TOKENS` and change codebase to use that everywhere.

### CODE REVIEW

File: kong/plugins/aws-lambda/handler.lua [hunk 1]
```diff
@@ -150,6 +153,10 @@ function AWSLambdaHandler:access(conf)
     ngx.status = res.status
   end
 
+  if singletons.configuration.enabled_headers[constants.HEADERS.VIA] then
+    ngx.header["Via"] = meta._SERVER_TOKENS
```
Comment (thibaultcha) [inline kong/plugins/aws-lambda/handler.lua]: We should also test this (or make it part of a separate commit)

File: kong/tools/responses.lua [hunk 2]
```diff
@@ -123,7 +123,18 @@ local function send_response(status_code)
...
+      if singletons.configuration.enabled_headers[constants.HEADERS.SERVER] then
+        ngx.header["Server"] = server_header
+
+      else
+        ngx.header["Server"] = nil
+      end
+
+    else
+      ngx.header["Server"] = server_header
+    end
```
Comment (thibaultcha) [inline kong/tools/responses.lua]: Ditto: probably belong to a separate commit (i.e. in the PR already opened for delayed response)

Comment (bungle) [reply kong/tools/responses.lua]: I think you wanted to comment about request-termination plugin changes here? I will make separate PR for the two plugins.

File: spec/01-unit/009-responses_spec.lua [hunk 3]
```diff
@@ -192,4 +192,33 @@ describe("Response helpers", function()
       assert.spy(s).was.called_with(ngx.ctx)
     end)
   end)
+
+  describe("server tokens", function()
+    it("are send by default", function()
```
Comment (hishamhm) [inline spec/01-unit/009-responses_spec.lua]: typo: `are sent by default` (likewise below)

**Comment by hishamhm** [comment]
I guess the first commit should be `refactor` instead of `feat` since this is an internal code reorganization without visible functionality change
